meta-mender-core/recipes-bsp/u-boot/patches/0002-Generic-boot-code-for-Mender.patch

```diff
@@ -166,7 +166,13 @@ index 0000000..705c39e
 +                                                                        \
 +    "mender_dtb_name=" MENDER_DTB_NAME "\0"                             \
 +                                                                        \
++    "mender_pre_setup_commands=" MENDER_PRE_SETUP_COMMANDS "\0"         \
++                                                                        \
```

Comment:
I think I would not include this in our patch, but rather encourage the user to patch it in themselves using `SRC_URI` and a patch file (for example using `CONFIG_EXTRA_ENV_SETTINGS`). This also saves us having to introduce a bitbake variable for it.

The calling code should still work.

Reply:
The motivation was to avoid more variables for things where there already is a mechanism. But maybe you're right, we are already adding some code for this, might as well add it all.

But then I would at least rename the variables to `MENDER_UBOOT_PRE_SETUP_COMMANDS` and `MENDER_UBOOT_POST_SETUP_COMMANDS` to keep them somewhat namespaced (like `MENDER_UBOOT_AUTO_CONFIGURE`.